client/galaxy/scripts/components/Workflow/Run/WorkflowRun.vue

```diff
@@ -10,7 +10,7 @@
             <workflow-run-success
                 v-else-if="invocations != null"
                 :invocations="invocations"
-                :workflowName="workflowName"
+                :workflow-name="workflowName"
```

Comment:
OK, obvious noob question: are `:workflowName` and `:workflow-name` equivalent ?

Reply:
Yep; html attribute names are case insensitive so kebab-case is preferred here; Vue automagically converts this.  https://vuejs.org/v2/style-guide/#Prop-name-casing-strongly-recommended